pkg/authentication/example_test.go

```diff
@@ -69,3 +72,146 @@ func ExampleApiClient_Octocat() {
 	//              ~~~~~~==~==~==~==~~~~~~
 	//                  :~==~==~==~==~~
 }
+
+func Example_ResponseHandlerForRateLimiting() {
```

Comment:
This function (currently with intentional malformed example suffix) shows using a Kiota abstraction for a response handler.